libs/components/forms/src/lib/modules/input-box/fixtures/input-box.component.fixture.html

```diff
@@ -603,4 +609,17 @@
   </sky-input-box>
 </div>
 
+<div class="input-not-wrapped">
+  <div class="sky-form-group">
+    <input
+      [autocomplete]="autocomplete"
```

Comment:
You'll need an example where `autocomplete` is not specified at all (see comment on the directive for why).

Reply:
Done. I also found that due to a logic mistake - the non-input box tests weren't quite working right. I resolved that.